package.json

```diff
@@ -5,58 +5,60 @@
   "private": true,
   "author": "BigCommerce",
   "license": "MIT",
+  "dependencies": {
+    "@bigcommerce/stencil-utils": "^1.0.7",
```

Comment:
webpack now complains about all of these deps being in devDeps and says to move them here